src/main/java/de/metas/ui/web/dashboard/json/JsonUserDashboardItemAddRequest.java

```diff
@@ -57,10 +57,17 @@
 
 	@AllArgsConstructor
 	@Getter
-	public static enum JSONWhen
+	public enum JSONWhen
 	{
 		now(null), lastWeek("P-7D");
 
 		private final String esTimeRangeEnd;
 	}
+
+	@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
aren't those requests coming from the same metasfresh code-version? because in that case i think there won't be unexpected properties..

Reply:
you're right, there's no need for ignoring those properties.